openproblems/tasks/dimensional_reduction/metrics/__init__.py

```diff
@@ -1 +1,4 @@
-##
+from .root_mean_square_error import compare_RMSE
+from .root_mean_square_error import plot_rmse
```

Comment:
We should only be importing metrics that return a float here. It looks like all but the first import are superfluous here.

Reply:
fixed, thanks